src/Entity/File.php

```diff
@@ -78,4 +78,47 @@ public function setDimensions(?array $dimensions): void
     {
         $this->dimensions = $dimensions;
     }
+
+    /**
+     * A simple shortcut to the image width.
+     * Similar to `$file->getDimensions()[0]`.
+     *
+     * @return int|null Returns `null` if dimensions array is itself null
+     */
+    public function getWidth(): ?int
+    {
+        return null !== $this->dimensions ? $this->dimensions[0] : null;
+    }
+
+    /**
+     * A simple shortcut to the image height.
+     * Similar to `$file->getDimensions()[1]`.
+     *
+     * @return int|null Returns `null` if dimensions array is itself null
+     */
+    public function getHeight(): ?int
+    {
+        return null !== $this->dimensions ? $this->dimensions[1] : null;
+    }
+
+    /**
+     * Format image dimensions for use with html (to avoid layout shifting).
+     *
+     * Usage in twig template:
+     * ```twig
+     * <img src="..." alt="..." {{ image.getHtmlDimensions }}>
+     * <!-- Will render: -->
+     * <img src="..." alt="..." width="..." height="...">
+     * ```
+     *
+     * @return string|null Returns `null` if dimensions array is itself null
+     */
+    public function getHtmlDimensions(): ?string
+    {
+        if (null !== $this->dimensions) {
+            return 'width='.$this->getWidth().' height='.$this->getHeight();
```

Comment:
Maybe a sprintf can be better than a triple concatenation. Moreover, this approach is forcing to add a space that can be useless. Maybe it's better to return an initial space (and change comment accordingly)

Reply:
I agree for `sprintf`.

But is the useless space in rendered Html very important?
Imo, drop it in twig doesn't looks very good in terms of readability:

```twig
<img src="..." alt="..."{{ image.htmlDimensions }}>
```
And btw, this function will only be used on `img` tags. So it can returns null, but it shouldn't, as it's surly an image which should have dimensions.

Reply:
let's keep the space, then